src/api/authed/payments/codes.js

```diff
@@ -0,0 +1,41 @@
+const Airbase = require("../../../airtable.js");
+const P2pMoney = require("./../../../p2p-money/p2p-money");
+
+// Consumes payment codes sent from donors/recipients via twilio
+// hands off to p2p money business logic to update payment status
+
+// POST params     --> is there somewhere we're supposed to enforce or document these?
+// - code: twilio expects this short alpha code is probably a valid payment code
+// - tel:  sender mobile #
```

Comment:
See comment -- nothing I can see in here is self-documenting re: what these requests should look like; anything else I can add, or anywhere we are documenting the API?